src/test/java/nl/tudelft/ewi/devhub/modules/MockedGitoliteGitServerModule.java

```diff
@@ -126,7 +126,6 @@ protected void createMockedMirrorsFolder() {
 	protected void createMockedRepositoriesFolder() {
 		String repositoriesPath = repositoriesFolder.toPath().toString() + "/";
 		when(configuration.getGitoliteBaseUrl()).thenReturn(repositoriesPath);
-		when(configuration.getRepositoriesDirectory()).thenReturn(repositoriesFolder);
```

Comment:
I have reverted it now. For some reason the tests were not failing when I ran them locally, so let's see if CI complains.